language/stdlib/CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing to the Move standard library
+
+This guide describes the process for adding, removing, and changing the Move modules and transaction scripts in the standard library.
+
+## Overview
+
+Every state change in the Libra blockchain occurs via executing a Move *transaction script* embedded in a [SignedTransaction](../../types/src/transaction/mod.rs). A transaction script invokes procedures of Move *modules* that update published *resources*. The Move standard library consists of:
+
+1. The [modules](modules/) published in the genesis transaction.
+2. The authorized [transaction scripts](transaction_scripts/) that can be included in a Libra transaction. A transaction with an unauthorized script will be discarded by validators.
+
+## Environment Setup
+
+Start by following the general Libra setup advice [here](../../CONTRIBUTING.md). Nothing else is strictly required, but you may want to consider a Move syntax highlighter for your editor (asking it to interpret `.move` files as Rust source is a decent start).
+
+<!-- TODO: editor-specific suggestions, bash aliases -->
+
+### Building
+
+Execute
+
+`cargo run`
+
+inside `stdlib` to compile all of the standard library modules, transaction scripts, and supporting Rust wrappers. It is important to do this before running any tests that exercise your change.
+
+### Testing
+
+Most tests for the standard library live [here](../move-lang/tests/functional) and can be run with `cargo test`.
```

Comment:
Not sure if it might be worth mentioning (or if it complicates things too much) but you do not need to build stdlib in order to have those changes tested in `move-lang/tests` (but you do in IR tests/e2e tests).

Reply:
I was planning not to mention this only because the recommended workflow below is to ensure that the stdlib builds correctly before you try to write tests (otherwise you might be confused about why e.g. your new procedure isn't found)... 

For IR tests this is indeed nasty and worth mentioning, but only Move VM/bytecode verifier contributors) should need to write those (never stdlib developers). We can mention that in the contribution guide for the VM/bytecode verifier once we have one.